src/controllers/auth.controller.js

```diff
@@ -29,7 +29,7 @@ class AuthController {
         email: req.body.email.trim(),
         password: BcryptService.hashPassword(req.body.password.trim())
       };
-      const createdUser = await UserService.addUser(userInput);
+      const createdUser = await UserService.addUser({ ...userInput, lineManagerId: 1 });
```

Comment:
@adesege currently, the feature to assign lineManager to users is being implemented by @gadishimwe and it is not yet deployed.So, that why i hard coded it because without lineManagerId, a trip can't be created.

Reply:
Implement this feature as it should then add a label that this PR is blocked by another PR.

Reply:
yes i removed the hardcoded stuff.